content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -510,6 +512,198 @@ const doWork = (parent, i) => {
 All other APIs behave the same when you use `sdk-trace-base` compared with the
 Node.js or Web SDKs.
 
+## Metrics
+
+To start [metrics](/docs/concepts/signals/metrics), you'll need to have an
+initialized `MeterProvider` that lets you create a `Meter`. `Meter`s let you
+create `Instrument`s that you can use to create different kinds of metrics.
+OpenTelemetry JavaScript currently supports the following `Instrument`s:
+
+* Counter, a synchronous instrument which supports non-negative increments
+* Asynchronous Counter, a asynchronous instrument which supports non-negative
+  increments
+* Histogram, a synchronous instrument which supports arbitrary values that are
+  statistically meaningful, such as histograms, summaries or percentile
+* Asynchronous Gauge, an asynchronous instrument which supports non-additive
+  values, such as room temperature
+* UpDownCounter, a synchronous instrument which supports increments and
+  decrements, such as number of active requests
+* Asynchronous UpDownCounter, an asynchronous instrument which supports
+  increments and decrements
+
+If a `MeterProvider` is not created either by an instrumentation library or
+manually, the OpenTelemetry APIs for tracing will use a no-op implementation and
+fail to generate data.
+
+### Initialize Metrics
+
+To initialize metrics, make sure you have the right packages installed:
+
+```shell
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/resources \
+  @opentelemetry/semantic-conventions \
+  @opentelemetry/sdk-metrics \
+  @opentelemetry/instrumentation
+```
+
+Next, create a separate `instrumentation.js` file that has all the SDK
+initialization code in it:
```

Comment:
where and how is this `instrumentation.js` used within the app? Is there something missing like `node -r ./instrumentation.js app.js` or a require statement for the app.js? 

I am wondering if tracing.js & instrumentation.js could be merged into one file?

Reply:
Done

Reply:
I don't like merging them into one file. In real-world instrumentation setup there's often even more configured here and it's nicer to be able to separate out the instrumentation setup goop from the rest of the app.